docs/BR.md

```diff
@@ -452,6 +455,8 @@ The script outputs:
 
 **Root Certificate**: The self-signed Certificate issued by the Root CA to identify itself and to facilitate verification of Certificates issued to its Subordinate CAs.
 
+**Short-lived Subscriber Certificate**: A server Certificate with a Validity Period that is less than ten (10) days (864,000 seconds). 
```

Comment:
In #402, @BenWilson-Mozilla commented:

```Can the Validity Period be defined as a shorter period, e.g. five days, 432,000 seconds?```

Open for discussion. 

10 days was in attempt to:
- align with existing timelines (e.g.,  CRL "nextUpdate") and implementations observed in some browsers  
- offer an attractive incentive for CAs to adopt automation. Reducing the period too far may result in limited adoption due to the impact of unforeseen outages or operational issues. IMO, a validity of 10 days is more "forgiving" of an unexpected 2-3 day service outage than a validity of 5 days).

What are your thoughts on a phased approach (sample below):
- Until 2026: 10 days
- Effective 1/1/26: 5 days

Reply:
@BenWilson-Mozilla - added a phased reduction in 6.3.2.

```Subscriber Certificates issued on or after 1 September 2020 SHOULD NOT have a Validity Period greater than 397 days and MUST NOT have a Validity Period greater than 398 days. To qualify as a Short-lived Subscriber Certificate, Subscriber Certificates issued after 15 January 2024, but prior to 15 January 2026, MUST NOT have a Valdity Period greater than 10 days. Short-lived Subscriber Certificates issued on or after 15 January 2026 MUST NOT have a Validity Period greater than 7 days.```

Reply:
Closing stale comments.